### ISSUE TITLE
App does not run in an emulator.

### ISSUE THREAD

**Issue by SebastianKurp** [Instabug/Instabug-React-Native#200]
### Steps to Reproduce the Problem
Open the app in an  android emulator

### Expected Behavior
The app loads and does not receive a red screen of death.

### Actual Behavior
Receives the red screen of death.

### Instabug integration code
```
            new RNInstabugReactnativePackage.Builder("Token goes here",MainApplication.this)
              .setInvocationEvent("shake")
              .setPrimaryColor("#1D82DC")
              .setFloatingEdge("left")
              .setFloatingButtonOffsetFromTop(250)
              .build(),
```

### SDK Version
8.0

### React Native, iOS and Android Versions
React-Native

### Device Model
Nexus 5X emulator

**Additional Info:**
My emulator is connected to the internet successfully, can open a browser and data is pulled from my db fine.

Info from the red-screen of death:
```
Could not invoke Instabug.networkLog

null

No value for responseBody
invoke
    JavaMethodWrapper.java:383
invoke
    JavaModuleWrapper.java:160
run
    NativeRunnable.java
handleCallback
    Handler.java:790
dispatchMessage
    Handler.java:99
dispatchMessage
    MessageQueueThreadHandler.java:29
loop
    Looper.java:164
run
    MessageQueueThreadImpl.java:192
run
    Thread.java:764
```

**Comment by Korazy** [comment]
Hey @SebastianKurp 

Thanks for pointing this out. We're working on a fix for it right now. I will keep you updated regarding its progress.

**Comment by Korazy** [comment]
Would you be able to tell me if you are making any network requests on app start. If so, would you be able to share with us that request?

**Comment by SebastianKurp** [comment]
The only request is one connecting to firebases auth service to verify the user's auth token if they have previously logged in.

**Comment by Korazy** [comment]
This is now fixed as of version 8.0.12. Please upgrade to it and feel free to reopen the issue if you run into it again :)
